configurations/ibm_huygens_sat_bmc0.json

```diff
@@ -0,0 +1,16 @@
+{
+    "Exposes": [
+        {
+            "AuthType": "None",
+            "Hostname": "10.0.2.100",
+            "Name": "satellite_controller",
+            "Port": 0,
+            "Type": "SatelliteController"
+        }
+    ],
+    "Name": "redundant bmc",
```

Comment:
nit: usually EM names are Capitalized.

Reply:
OK, I’ll modify it to match the style of EM... Just followed like other components in OpenBMC.